src/pages/guildverse.tsx

```diff
@@ -0,0 +1,106 @@
+import { SimpleGrid } from "@chakra-ui/react"
+import BrainCard from "components/brain/BrainCard"
+import FilterSelect, { FilterOption } from "components/brain/FilterSelect"
+import Layout from "components/common/Layout"
+import LinkPreviewHead from "components/common/LinkPreviewHead"
+import CategorySection from "components/explorer/CategorySection"
+import SearchBar from "components/explorer/SearchBar"
+import { GetServerSideProps } from "next"
+import { useMemo, useState } from "react"
+import { BrainCardData } from "types"
+
+type Props = {
+  cards: BrainCardData[]
+}
+
+export const FILTER_OPTIONS: Array<FilterOption> = [
```

Comment:
The labels could start with uppercase letters, they'd look better that way in my opinion.